index.html

```diff
@@ -22,6 +22,12 @@
     <link href="assets/css/stylesheet.css" rel="stylesheet">
     <script src="assets/js/core.js"></script>
 
+    <!-- Bootstrap modal -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css">
```

Comment:
we already import bootstrap [above](https://github.com/dinuand/sites_template/blob/master/index.html#L11-L13)